drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -30,7 +30,8 @@ class Hmis::Hud::Client < Hmis::Hud::Base
   has_many :files, class_name: '::Hmis::File', dependent: :destroy, inverse_of: :client
   has_many :current_living_situations, through: :enrollments
   has_many :hmis_services, through: :enrollments # All services (HUD and Custom)
-  has_many :external_ids, class_name: 'HmisExternalApis::ExternalId', foreign_key: :source_id
+  has_many :external_ids, class_name: 'HmisExternalApis::ExternalId', as: :source
+  has_many :external_referral_household_members, class_name: 'HmisExternalApis::ReferralHouseholdMember', dependent: :destroy, inverse_of: :client
```

Comment:
These relationships should be injected via extension from the `hmis_external_apis`

Reply:
Can you say more about this @eanders? If this desire is to have no dependencies from `hmis` on `hmis_external_apis`, I don't think that's feasible. There will be several places where we need to call methods in the external api driver directly: Referral Request form processor, Client form processor (for MCI), initiate unit availability update, etc.

Reply:
Sure, this isn't an attempt to completely decouple the drivers, just that the driver providing the has_many should be injecting it.  Calling something can be easily wrapped with an `if driver is installed` later, but this moves the definition of the functionality into the driver where the models live.

Reply:
Oh gotcha. Thanks for the explanation!